operator/hack/csv-generate.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+set -e
+
+export GOROOT=$(go env GOROOT)
+
+#PREV="0.0.1"
+LATEST="0.0.1"
+
+if [[ -z "$CSV_VERSION" ]]; then
+  CSV_VERSION=$LATEST
+  REPLACES_CSV_VERSION=$PREV
+fi
+
+PACKAGE_NAME="observatorium-operator"
+
+PACKAGE_DIR="${CATALOG_DIR}/${PACKAGE_NAME}"
+CSV_DIR="${PACKAGE_DIR}/${CSV_VERSION}"
+CSV_FILE="$CSV_DIR/${PACKAGE_NAME}.v${CSV_VERSION}.clusterserviceversion.yaml"
+
+OUT_DIR="_output/olm-catalog"
+OUT_CSV_DIR="${OUT_DIR}/${PACKAGE_NAME}/${CSV_VERSION}"
+OUT_CSV_FILE="${OUT_CSV_DIR}/${PACKAGE_NAME}.v${CSV_VERSION}.clusterserviceversion.yaml"
+
+ANNOTATIONS_FILE="${DEPLOY_DIR}/csv/annotations.json"
+MAINTAINERS_FILE="${DEPLOY_DIR}/csv/maintainers.json"
+
+EXTRA_ANNOTATIONS=""
+MAINTAINERS=""
+
+if [ -n "$DESCRIPTION_FILE" ]; then
+	DESCRIPTION="-description-from=$DESCRIPTION_FILE"
+fi
+if [ -n "$MAINTAINERS_FILE" ]; then
+	MAINTAINERS="-maintainers-from=$MAINTAINERS_FILE"
+fi
+if [ -n "$ANNOTATIONS_FILE" ]; then
+	EXTRA_ANNOTATIONS="-annotations-from=$ANNOTATIONS_FILE"
+fi
+
+clean_package() {
+#	mkdir -p "$CSV_DIR"
+	rm -rf "$OUT_DIR"
+	mkdir -p "$OUT_CSV_DIR"
+}
+
+if ! [[ "$CSV_VERSION" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
```

Comment:
This is not a correct server regex, consider taking one from the semver website

Reply:
right, will use this then: https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string